R/transition_risk_profile.R

```diff
@@ -101,8 +101,11 @@ transition_risk_profile <- function(emissions_profile,
     polish_best_case_worst_case() |>
     polish_transition_risk_at_product_level()
 
+  risk_categories_at_company_level <- add_transition_risk_category_at_company_level(product)
+
   company <- transition_risk_scores |>
     unnest_company() |>
+    join_risk_categories_at_company_level(risk_categories_at_company_level) |>
     polish_transition_risk_at_company_level()
```

Comment:
It would be nice to structure this chunk of code so that we create `product` and `company` with similar pipes. That may read more easily, but also may have the beneficial side effect of forcing us to find similarities that may result in more reusable code.

Reply:
@maurolepore Please review the code again to see if it meets this expectation of yours! Thanks!